src/engraving/dom/keysig.cpp

```diff
@@ -296,4 +296,13 @@ String KeySig::accessibleInfo() const
     String keySigType = TConv::translatedUserName(key(), isAtonal(), isCustom());
     return String(u"%1: %2").arg(EngravingItem::accessibleInfo(), keySigType);
 }
+
+//---------------------------------------------------------
+//   subtypeUserName
+//---------------------------------------------------------
+
+String KeySig::translatedSubtypeUserName() const
```

Comment:
And here

Reply:
The `TConv::userName` method for key signatures returns `char*` instead of `TranslatableString` like the occurences outside of keysigs. Should this be changed to a `TranslatableString`?

Reply:
Yes, makes sense to me!